src/views/Wallets.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <div :class="classes.root">
+    <app-toolbar-centered app :title="$t('options.wallets_list')" :show-back="true" flat fixed />
+
+    <v-container fluid class="px-0 container--with-app-toolbar">
+      <v-row justify="center" no-gutters>
+        <container>
+          <WalletsSearchInput @change="searchChanged"></WalletsSearchInput>
+          <v-list lines="one">
+            <draggable
+              class="list-group"
+              v-model="filteredWallets"
+              v-bind="dragOptions"
+              handle=".handle"
+              @start="isDragging = true"
+              @end="isDragging = false"
+              item-key="cryptoName"
+            >
+              <template #item="{ element }">
+                <WalletsListItem :wallet="element" :search="search"></WalletsListItem>
+              </template>
+            </draggable>
+          </v-list>
+          <v-row
+            :class="`${classes.root}__review`"
+            align="center"
+            justify="space-between"
+            no-gutters
+          >
+            <v-spacer></v-spacer>
+            <WalletResetDialog></WalletResetDialog
+          ></v-row>
+        </container>
+      </v-row>
+    </v-container>
+  </div>
+</template>
+
+<script lang="ts">
+import draggable from 'vuedraggable'
+import AppToolbarCentered from '@/components/AppToolbarCentered.vue'
+import { computed, defineComponent, onBeforeUnmount, onMounted, ref } from 'vue'
+import { CryptosInfo, CryptoSymbol, isErc20 } from '@/lib/constants'
+import { useStore } from 'vuex'
+import WalletsSearchInput from '@/components/wallets/WalletsSearchInput.vue'
+import WalletsListItem from '@/components/wallets/WalletsListItem.vue'
+import WalletResetDialog from '@/components/wallets/WalletResetDialog.vue'
+import { Timer } from 'web3-utils'
+
+const BALANCE_UPDATE_INTERVAL_MS = 30000
+
+const className = 'wallets-view'
+const classes = {
+  root: className
+}
+
+const dragOptions = {
+  animation: 200,
+  disabled: false,
+  ghostClass: 'ghost',
+  group: 'description'
+}
+
+type OrderedWalletSymbol = {
+  isVisible: boolean
+  symbol: CryptoSymbol
+}
+
+type Wallet = {
+  erc20?: boolean
+  balance?: number
+  cryptoName?: string
+  currentRate?: number
+  isVisible: boolean
+  key?: string
+  rate?: number
+  symbol: CryptoSymbol
+  type?: string
+}
+
+export default defineComponent({
+  components: {
+    WalletResetDialog,
+    WalletsListItem,
+    WalletsSearchInput,
+    AppToolbarCentered,
+    draggable
+  },
+  setup() {
+    const store = useStore()
+
+    const isDragging = ref(false)
+    const search = ref('')
+
+    const orderedAllWalletSymbols = computed(() => {
```

Comment:
```suggestion
    const orderedAllWalletSymbols = computed<CoinSymbol[]>(() => {
```

You can type the value using generics.

Reply:
After, remove the type `OrderedWalletSymbol` on line 100